tjmonopix2/system/tjmono2_rx.py

```diff
@@ -10,8 +10,13 @@
 
 
 class tjmono2_rx(RegisterHardwareLayer):
-    '''TJ-Monopix2 receiver controller interface for tjmono2_rx FPGA module
-    '''
+    """TJ-Monopix2 receiver controller interface for tjmono2_rx FPGA module
+
+    Parameters
+    ----------
+    RegisterHardwareLayer : class
+        Allows register read write operations
+    """
```

Comment:
As for the other drivers: this implements the configuration of the FPGA receiver module. Or use the term controller in all other instances of this type to keep it consistent.